tests/model_fields/test_choiceenum.py

```diff
@@ -0,0 +1,282 @@
+import datetime
+import decimal
+import enum
+import ipaddress
+import uuid
+
+from django.db import models
+from django.db.models.enums import ChoiceEnumMeta
+from django.test import SimpleTestCase, TestCase
+from django.utils.functional import Promise
+from django.utils.translation import gettext_lazy as _
+
+from .models import Foo
+
+
+class Suit(models.ChoiceIntEnum):
+    DIAMOND = 1, _('Diamond')
+    SPADE = 2, _('Spade')
+    HEART = 3, _('Heart')
+    CLUB = 4, _('Club')
+
+
+class YearInSchool(models.ChoiceStrEnum):
+    FRESHMAN = 'FR', _('Freshman')
```

Comment:
I don't think that we need to use `gettext_lazy` in all enums.

Reply:
Will leave for `Suit` and `YearInSchool` as this is explicitly tested for each of `ChoiceIntEnum` and `ChoiceStrEnum`, but will remove all the others.

Reply:
Will leave for `Suit` and `YearInSchool` as this is explicitly tested for each of `ChoiceIntEnum` and `ChoiceStrEnum`, but will remove all the others.